tasks/2017/interactive-bots.md

```diff
@@ -35,92 +35,56 @@ messages and can be integrated with various APIs.
 There are four types of tasks in this category, each corresponding to one of
 the Task Types below.
 
-### Task Type A: Learn about interactive bots by running the followup bot
+### Task Type A: Learn about interactive bots by running the 'helloworld' bot
 
-Follow the tutorial below to test run the followup bot on your Zulip
-instance (either local or in your droplet).
+Follow the tutorial below to test run the 'helloworld' bot on your Zulip
+instance (either local or in your droplet).  As you're going through the tutorial,
+take notes on where you got stuck or needed to look up terminology.
 
-* Run the Zulip server and log in the browser as Cordelia.
+Refer to the detailed description on how to [run a zulip bot](https://chat.zulip.org/api/running-bots).
 
-* Find your user's API key just below the bots section in the settings page.
-This is how Zulip knows the request is from an authorized user.
+Follow the steps below to run "helloworld" bot:
 
-  You can also create a new bot for a user in settings - it will have its
-  own API key and email. Remember to subscribe it to the streams you want to
-  use the bot in.
+* Create a bot with the name `<your-name>_helloworld_bot`.
+(For details refer to the second step [here](https://chat.zulip.org/api/running-bots))
 
-* Download your bot's `zuliprc` file, by clicking on the green button, and
-save it as `~/zuliprc-local`.
+* Since we aim to run a "helloworld" bot, replace <my_bot> with "helloworld" while [running a bot](https://chat.zulip.org/api/running-bots).
 
-  Alternatively, you can create this file with credentials:
+* Use `Manage Streams` to subscribe your bot (`<your-name>_helloworld_bot`) to any stream
```

Comment:
We don't need to subscribe bots to streams anymore, so this step can be removed.

Reply:
Oh is it? That's interesting.
So, the bots get subscribed to all the streams or the ones that the owner of the bot is subscribed to?

Reply:
No, the change is that bots don't need to be subscriobed to a stream to receive @-mentions anymore.